Gems/LmbrCentral/Code/Source/Shape/EditorSplineComponentMode.cpp

```diff
@@ -68,6 +68,11 @@ namespace LmbrCentral
         AzToolsFramework::EditorVertexSelectionActionManagement::BindEditorVertexSelectionActionsToMenus();
     }
 
+    void EditorSplineComponentMode::PostActionManagerRegistration()
+    {
+        AzToolsFramework::EditorVertexSelectionActionManagement::DisableComponentModeEndOnVertexSelection();
+    }
```

Comment:
question: Could this maybe go in EditorVertexSelection instead of the Component Mode so this works for PolygonPrism and Spline and doesn't need to be duplicated in both places? As I think it'll be needed there too